doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -468,10 +468,23 @@ Supported settings
 
     See :ref:`automatic-memory-scaling` for further information and examples.
 
+.. _bps_number_of_retries:
+
 **numberOfRetries**, optional
     The maximum number of retries allowed for a job (must be non-negative).
     The default value is ``None`` meaning that the job will be run only once.
 
+.. _bps_retry_unless_exit:
+
+**retryUnlessExit**, optional
+    Non-zero exit code(s) for which a job should not be retried when
+    ``numberOfRetries`` is set.  The biggest use case is to allow jobs
```

Comment:
>The biggest use case is to allow jobs exceeding requested memory to be automatically retried, but not jobs failing due to science issue.

Maybe we could rewrite this sentence as

> It can be used to prevent WMS from retrying jobs failing due to science issues while letting it to retry jobs exceeding requested memory.